cms/templates/base.html

```diff
@@ -1,11 +1,12 @@
 ## coding=utf-8
 <%namespace name='static' file='static_content.html'/>
 <%!
-from django.utils.translation import ugettext as _
+from openedx.core.djangolib.markup import ugettext as _
 from openedx.core.djangolib.js_utils import (
     dump_js_escaped_json, js_escaped_string
 )
 %>
+<%page expression_filter="h"/>
```

Comment:
Reviewers: calling attention to this line which is what adds html escaping to all variables on this page by default.